docs/UserGuide.md

```diff
@@ -108,6 +108,9 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
+* After using commands like `find`, the list may no longer display all contacts.
+* Use `list` to revert the list to the full list of contacts in the order they were added.
```

Comment:
Looks good but it's probably better to put this under `find` command, and here just say "see `find` command's documentation (⟵ link to corresponding section) to know in which case this command is useful"